torch/_inductor/mkldnn_lowerings.py

```diff
@@ -551,26 +565,73 @@ def qlinear_unary(
                 )
                 if (
                     isinstance(
-                        ir.InputsKernel.unwrap_storage_for_input(x_zp),
-                        ir.ConstantBuffer,
+                        ir.InputsKernel.unwrap_storage_for_input(w_zp),
+                        ir.ComputedBuffer,
```

Comment:
I think for this DA8W8, the `w_zp` should still be a `ConstantBuffer`, isn't it?

Reply:
@Xia-Weiwen created dummy `w_zp` for the smoothquant pattern, and the smaller da8w8 pattern I added for sym act, sym wgt case also leverages the same lowering, and it'd be `ComputedBuffer` in that case

Reply:
It's better we only focus on `WOQ DA8W8` Template in this PR, Can we leave the `smoothquant` Template support in the following PR?

Reply:
da8w8 implies that the activation is also quantized.

> It's better we only focus on `WOQ DA8W8` Template in this PR, Can we leave the `smoothquant` Template support in the following PR

The torchao smoothquant pattern with 2D activations is same as the torchao da8w8 sym act, sym wgt pattern for 2D activations